[
    {
        "content": "<p>You remember that incredible bug where a specific line was getting an error? Well, it seems like I'm onto another one. On <strong>line 282</strong>, around <strong>column 14</strong>, I get an <code>unexpected end of string</code>, whatever the code here actually is.<br>\nUnfortunately, I can't reproduce it. Restarting the Lean server made it go away.</p>",
        "id": 249076264,
        "sender_full_name": "Yaël Dillies",
        "timestamp": 1628666857
    },
    {
        "content": "<p><a href=\"https://github.com/leanprover-community/lean/issues/468\">lean#468</a> ?</p>",
        "id": 249099590,
        "sender_full_name": "Shing Tak Lam",
        "timestamp": 1628684901
    },
    {
        "content": "<p>Oh yeah, sounds like it!</p>",
        "id": 249099734,
        "sender_full_name": "Yaël Dillies",
        "timestamp": 1628684988
    }
]